cli/src/commands/git.rs

```diff
@@ -220,6 +220,9 @@ pub struct GitPushArgs {
     /// correspond to missing local branches.
     #[arg(long)]
     deleted: bool,
+    /// Push branches with commits that contain empty descriptions
```

Comment:
It sounds a bit like this would find commits with empty descriptions and push them. Maybe it's clear enough with the `--allow` part of the flag name, but maybe it's still better to say "Allow pushing commits with empty description" or something like that? I don't think the "branches" in the description really helps either (which is why I didn't include it in my proposal).

Reply:
Yeah, I agree that adding "allow" is a good idea. I wasn't sure about the terminology, but I think referring to commits instead of branches might be better.

Something like "Allow pushing commits with empty descriptions."